screeps-game-api/src/constants.rs

```diff
@@ -14,13 +14,12 @@
 //!
 //! [the game constants]: https://github.com/screeps/common/blob/master/lib/constants.js
 
-use log::error;
 use num_derive::FromPrimitive;
 use num_traits::FromPrimitive;
 #[allow(unused_imports)]
 use serde::{Deserialize, Serialize};
 use serde_repr::{Deserialize_repr, Serialize_repr};
-use stdweb::{Number, Reference, Value};
+use stdweb::{Number, Reference, Value, __js_deserializable_serde_boilerplate, js_deserializable};
```

Comment:
if you want to avoid weird macro imports all over the place, feel free to use `crate::macros::*` which exposes those macros.